test/kinematic/potential_energy_from_mass_and_height_test.py

```diff
@@ -0,0 +1,54 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+
+from symplyphysics import (
+    units, convert_to, SI, errors
+)
+from symplyphysics.laws.kinematic import potential_energy_from_mass_and_height as potential_energy
+
+@fixture
+def test_args():
+    m = units.Quantity('m')
+    SI.set_quantity_dimension(m, units.mass)
+    SI.set_quantity_scale_factor(m, 1 * units.kilogram)
+    g = units.Quantity('g')
+    SI.set_quantity_dimension(g, units.acceleration)
+    SI.set_quantity_scale_factor(g, 9.82 * units.meter / units.second**2)
+    h = units.Quantity('h')
+    SI.set_quantity_dimension(h, units.length)
+    SI.set_quantity_scale_factor(h, 1 * units.meter)
+    Args = namedtuple('Args', ['m', 'g', 'h'])
+    return Args(m=m, g=g, h=h)
+
+def test_basic_energy(test_args):
+    result = potential_energy.calculate_potential_energy(test_args.m, test_args.g, test_args.h)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.energy)
+    result_energy = convert_to(result, units.joule).subs(units.joule, 1).evalf(3)
+    assert result_energy == approx(9.82, 0.01)
```

Comment:
It seems you multiplied all values and put the result in the assertion. This kind of test does not actually test anything. I suggest to google for some real values that you can use in your test and verify it's correctness.